packages/vite/src/node/preview.ts

```diff
@@ -186,9 +189,25 @@ export async function preview(
 
```

Comment:
I'm not quite sure about this one. It would be a big breaking change I think for those expecting the base not removed in preview, and I think it's a bit late to make that change now 🤔